src/app/healthyrelationships/healthyrelationships.page.html

```diff
@@ -23,137 +23,134 @@
 
   <div *ngIf="subTab==='healthy'">
     <h1 style="font-size: 18px;">
-      Healthy, whether it's friendship or romantic
+      Signs of a Healthy Relationship
     </h1>
+    <h3 style="text-align: center; font-size: 14px;">
+      (friendship or romantic)
+    </h3>
     <ion-slides pager="true" [options]="slideOpts">
       <ion-slide>
-        <ion-card>
-          <img
-            src="https://static1.squarespace.com/static/58ebeae859cc68b20f410b26/5a281fa5e2c4838eb6e38450/5a302bb5419202a08e5e0347/1513106439847/listen.jpg?format=1500w"
-          />
+        <ion-card class="slide-card">
+          <img src="assets/images/healthy/listen.jpg" />
           <ion-card-header>
-            <ion-card-title>Listens to each other</ion-card-title>
+            <ion-card-title>Listen to each other at all times</ion-card-title>
           </ion-card-header>
         </ion-card>
+        <div class="slide-footer">1 of 11</div>
```

Comment:
I added this little thing that sticks to the bottom in order to let the user know how many slides there are to go through.